### ISSUE TITLE
The externals downloader has big issues.

### ISSUE THREAD

**Issue by fotisandstuff** [plugdata-team/plugdata#1102]
Hello!

I used the nightly that came out yesterday and the "find externals" menu drove me nuts. First, when I close plugdata and then open it again, it frequently forgets which externals have been downloaded and installed, even when I add them to path. Second, downloading some externals causes plugdata to crash (some even refuse to start downloading, such as that "plaintofia" or whatever it was called). I used the standalone version on Windows 10 x64.

**Comment by timothyschoen** [comment]
Thanks for reporting, I've heard this issue before but I thought it was solved. Apparently not!

Unfortunately I've never been able to reproduce this. I wonder, are you using an administrator account? I've heard someone suggest that it might not work when not using an admin account. if that's the case, that could give me a clue about what might be causing this.

If you are using an admin account, I guess I'll try installing planifolia a couple of times and restarting plugdata, to see if I can finally reproduce this.

**Comment by fotisandstuff** [comment]
Yes, I am using an admin account. I wonder if this can be fixed if I run plugdata with the "run as administrator" option.

**Comment by timothyschoen** [comment]
I'm trying to reproduce it, but still nothing. I'm testing on Windows 11 (x64) as that's all I have installed currently. I tried installing a bunch of externals and then restarting plugdata many times but they won't disappear. Are there any other steps I can try? Or any specific externals that seem to be affected?


I can confirm that planifolia doesn't want to install, I'm not completely sure why so I'll check that out.

**Comment by fotisandstuff** [comment]
The problem still persists with the latest nightly. I tried downloading the first 4-5 things from deken. After closing and opening PD a couple of times, they disappear from the Installed section. :(

**Comment by fotisandstuff** [comment]
Can confirm it still happens with 0.9.0.
